packages/protocol/contracts/governance/Governance.sol

```diff
@@ -46,9 +46,13 @@ contract Governance is
   }
 
   struct VoteRecord {
```

Comment:
what happens if this gets updated in the middle of a governance vote?

Reply:
Since votes themselves are being stored in Proposals.sol they are not affected. Only possible time when they are affected is when user will want to revote/revoke votes on already voted (before update) proposal. For this case there are two if branches in both voteInternal and revokeVotes with comment `// backward compatibility for transition period - this should be deleted later on`